### PR TITLE
dont append value if unit === none

### PR DESCRIPTION
**Pull request by dtaylor7**

### CODE REVIEW

**Review by AhdabNasir — DISMISSED** [review]

File: helpers/transitionReadinessData.js [hunk 1]
```diff
@@ -282,7 +282,10 @@ const mapEntityChildren = (allEntities, entity) => {
     entityFieldMap[entityFieldEntry.categoryField.name] = entityFieldEntry.value;
   }, {});
 
-  if(entityFieldMap.groupDescription && entityFieldMap.commentsOnly !== 'Yes') {
+  const hasGroupDescription = !!entityFieldMap.groupDescription;
+  const shouldAppendValue = entityFieldMap.unit !== 'none';
+
+  if(hasGroupDescription && shouldAppendValue) {
     if(entityFieldMap.unit === '%') {
       entityFieldMap.name = `${entityFieldMap.groupDescription}: ${entityFieldMap.value}%`;
```
Comment (gds-gharper) [inline helpers/transitionReadinessData.js:290]: I think these are worth a comment, for anyone else browsing the code

Comment (dtaylor7) [reply helpers/transitionReadinessData.js:290]: i thaught the names of the vars were comment enough? should I elaborate further?

Comment (gds-gharper) [reply helpers/transitionReadinessData.js:290]: More why we're using magic strings like "none" and not just looking for an empty string

Comment (dtaylor7) [reply helpers/transitionReadinessData.js]: i dont know the exact reason behind that, but I have added some comments so the developer is aware what should happen if the unit is "none"

**Review by gds-gharper — APPROVED** [review]

**Pull request merged**